Examples/iOSSwiftUI/ContentView.swift

```diff
@@ -1,10 +1,36 @@
 import HaishinKit
+import AVFoundation
+import VideoToolbox
 import SwiftUI
 
 struct ContentView: View {
+    @ObservedObject var viewModel = ViewModel()
+    
+    private var lfView: MTHKSwiftUiView!
+    private var menuView: MenuView!
+    
+    init() {
+        viewModel.config()
+        lfView = MTHKSwiftUiView(rtmpStream: $viewModel.rtmpStream)
+        menuView = MenuView(viewModel: viewModel)
```

Comment:
The problem with doing this is that the init can pass multiple times, especially when using native navigation with SwiftUI. Struct can be instantiate multiple time